src/ImageSharp.Drawing/Brushes/ImageBrush{TColor}.cs

```diff
@@ -86,7 +86,7 @@ public ImageBrushApplicator(IImageBase<TColor> image, RectangleF region)
             /// <returns>
             /// The color
             /// </returns>
-            public TColor GetColor(Vector2 point)
+            public override TColor GetColor(Vector2 point)
```

Comment:
+ I think it makes no sense to use `Vector2` here. For brushes you are casting int-s to cast them back to float-s. It's expensive.
+ Need to profile code, but I'm quite sure we can make significant performance gain relatively cheaply by having a batched "big brother" for `GetColor`:
```
public virtual void PopulateRow(int y, int fromX, int toX, Span<TColor> destination)
```
Maybe this story is about a future PR.